packages/preview/vita/0.1.0/typst.toml

```
@@ -0,0 +1,9 @@
+[package]
+name = "vita"
+version = "0.1.0"
+entrypoint = "vita.typ"
+authors = ["Joey Carpinelli"]
+license = "MIT"
+description = "Modern résumé templates that separate content and style."
+repository = "https://github.com/cadojo/modern-resumes"
```

I think this needs to vita here as well, right? :)

```suggestion
repository = "https://github.com/cadojo/vita"
```

---

Sure does! Thanks :)